### DISCUSSION TITLE
flask not working at all

### DISCUSSION THREAD

**Discussion by Adamyoboy** [pallets/flask#4499] [Q&A]
I put `import flask from Flask` even though I did the pip install the module isn't working for some reason?

**Comment by OliverSparrow** [comment]
The ```import flask``` statement is case-sensitive. Try the following import statement instead:

 ```from flask import Flask```

Hope this helps!

**Comment by srkds** [comment]
Try 👇  importing using the following method
```py
from flask import Flask
```
